### ISSUE TITLE
--help for migrate doesn't print anything.

### ISSUE THREAD

**Issue by Leopere** [JordanKnott/taskcafe#25]
Trying to follow the instructions deploying this to a Docker Swarm and struggling to get `./citadel migrate` to run.

```bash
~ # ./citadel migrate --help
Run the database schema migrations

Usage:
  citadel migrate [flags]

Flags:
  -h, --help   help for migrate
```

**Comment by Leopere** [comment]
Can this migration not be smarter like can it just detect if the migration hasn't been done and then just do it?  Because having to hack around like this is a pain in the ass.  Ideally, just migrate the database automatically if there's nothing detected in the tables so that the app can work out of the box.

**Comment by JordanKnott** [comment]
The `--help` flag literally just prints the usage. Not very helpful I know but that's what it does right now. 

As far as migrations running automatically, I'm hesitant to do this as I prefer them to be separate steps. Perhaps I will change this in the future. 

If you run this command, `docker-compose -p citadel -f docker-compose.yml -f docker-compose.migrate.yml run --rm migrate`, what is the output?

**Comment by Leopere** [comment]
That’s not a useful command in a Docker swarm deployment sadly but i
understood how to translate it. The less mocking about I’m required to do
in a container command line the better really.

**Comment by JordanKnott** [comment]
Oh, docker swarm. I've never used that so I'm not much help. 

I might add a way to have migrations run on app start-up if an env flag is set ( which would be useful for me during dev) so that its at least an option for those that want it.

**Comment by Leopere** [comment]
I’ll take it! Can it be used momentarily in deployment and have a bit of
smarts in it for the sake of not clobbering the preexisting data?

**Comment by JordanKnott** [comment]
Hmm, I'm not sure what you mean? 

Migration steps only run if they haven't been ran already.

**Comment by Leopere** [comment]
Then why not just run the migration steps always?

**Comment by JordanKnott** [comment]
Because sometimes you want to run migrations as separate step (such as if you have multiple web instances) however I do think at least having the ability to run migrations on startup would be helpful (especially in a dev env)

Closed in favor of #29
